handbook/benefits.md

```diff
@@ -53,26 +78,69 @@ The following stipend amounts have been developed to help cover some of the cost
 
 ### **Retirement & Pension Plans**
 
 As previously mentioned we look to target locally relevant and competitive benefit options and retirement and pension plans fall under the same philosophy. We are currently working towards getting each required Pension scheme and retirement plan solidified. More details will be forthcoming.
 
-## [Healthcare Benefits](https://www.notion.so/dbtlabs/US-Healthcare-Benefits-Overview-650cf65d9612445a8d7ce41e936b307f?pvs=4)
+## [Healthcare Benefits (US)](https://www.notion.so/dbtlabs/US-Healthcare-Benefits-Overview-650cf65d9612445a8d7ce41e936b307f?pvs=4)
+
+### [Medical Plans](https://www.notion.so/dbtlabs/US-Healthcare-Benefits-Overview-650cf65d9612445a8d7ce41e936b307f?pvs=4)
+
+We provide medical, dental and vision benefits to employees and their dependents as well as FSA, DCFSA, and HSA options. Health benefits are paid in part by the company. The remainder of the costs is the employee’s responsibility
 
 Learn more about our medical plans [here](https://app.strivebenefits.com/dbt/2022-2023-medical?country=us).
 
 New hires may enroll themselves and dependents in health benefits online through ADP within the first 30 days of employment. There is a strict 30 day enrollment window. After benefit elections are submitted, enrollments go into effect the first day of employment (your start date) at dbt Labs.
 
 To enroll in benefits as a new hire, existing employee during open enrollment or at a time of a qualifying life event, follow our [guide](https://www.notion.so/dbtlabs/US-Health-Benefit-Enrollment-Guide-8132d651bdc54fddb8c3c6251701d5d5?pvs=4). Note: Employees who currently have a High Deductible Health Plan coverage selection are able to adjust their HSA contributions throughout the year. This is the only benefit adjustment that does not require a Qualifying Life or Work Event.
 
-## Stipends
+### Disability and Life Insurance
+We provide long term and short term disability as well as life insurance coverage to all US-based employees at no cost. Voluntary, additional, life insurance coverage for employees and dependants is available at the expense of the employee. Learn more:
+- [Life Insurance](https://app.strivebenefits.com/dbt/unum-life-and-add?country=us)
+- [Disability](https://app.strivebenefits.com/dbt/unum-disability?country=us)
+
+### US Employees enrolled in our health insurance are eligible for the following benefits as well:
+
+#### [Health Related Travel Benefit (US)](https://www.notion.so/dbtlabs/Health-Related-Travel-Benefit-1b3541b3eec3444d883de4ecf601dd9f?pvs=4)
+
+📌 We believe that it’s imperative that our team members have access to quality healthcare that they need, no matter where they live. Therefore, dbt Labs will reimburse up to $4,000 in travel expenses annually[^1] for any U.S. team member (including their dependent(s)) who requires healthcare that is not available in their state of residence.
```

Comment:
I think the * after "annually" in the notion doc showed up as a footer call out here. [^1] It's a link in the page that doesn't go anywhere so should be review.